src/components/ContactPolicy/ContactPolicy.test.js

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
-import { cleanup, fireEvent, render, waitFor } from '@testing-library/react';
-import { act } from 'react-dom/test-utils';
+import {
+  cleanup,
```

Comment:
Just to clarify. Some testing frameworks already cleanup automatically, including jest and mocha